service/src/main/kotlin/io/provenance/explorer/domain/entities/ExplorerCache.kt

```diff
@@ -328,11 +329,11 @@ class PulseCacheRecord(id: EntityID<Int>) : IntEntity(id) {
         fun upsert(date: LocalDate, type: PulseCacheType, data: PulseMetric, subtype: String? = null) = transaction {
             findByDateAndType(date, type, subtype)?.apply {
                 this.data = data
-                this.updatedTimestamp = LocalDateTime.now()
+                this.updatedTimestamp = LocalDateTime.now(ZoneOffset.UTC)
```

Comment:
this is typically set at the server level, and then jvm, but since I don't see the consistency in the codebase, this is probably prudent.